entrypoint.sh

```diff
@@ -70,7 +70,7 @@ echo_info "DESTINATION_BRANCH=$DESTINATION_BRANCH"
 # Determine repository url
 if [[ -z "$INPUT_DESTINATION_REPOSITORY" ]]; then
   # Try to query local repository's remote url if INPUT_DESTINATION_REPOSITORY is null
-  local_origin=$(git remote get-url origin)
+  local_origin=$(git remote get-url origin | sed -E 's/:([^\/])/\/\1/g' | sed -e 's/ssh\/\/\///g' | sed -e 's/git@/https:\/\//g')
```

Comment:
Thanks for the pr! Could you explain why we're using 3 sed?

What do you think about one Regex replace like https://gist.github.com/rsp/5ffd2c1f4e6278c58d47#file-git-change-url-L43

Reply:
It was something I picked up on SO and rushed to fix the issue. But ended up writing a much better sed script which lead to some cleanup as well. Changes were pushed and tested.